### PR TITLE
Update LoRa Cloud integration docs

### LINKED ISSUE #944: Update LoRa Cloud Geolocation Service integration support with TTS
#### Summary

The [LoRa Cloud Geolocation](https://www.thethingsindustries.com/docs/integrations/lora-cloud/geolocation/) Services integration has been deprecated (by Semtech) and cannot be used with the modem as per the context [here](https://thethingsindustries.atlassian.net/browse/TTS-4825?focusedCommentId=47713). Hence, the LoRa Cloud Geolocation integration process in the TTS documentation needs to be updated.

#### Why do we need this?

To assist users with the updated instructions on LoRa Cloud Geolocation Services integration in TTS.

#### What is already there? What do you see now?

Instructions on [LoRa Cloud Geolocation](https://www.thethingsindustries.com/docs/integrations/lora-cloud/geolocation/) integration with TTS.

#### What is missing? What do you want to see?

Updated instructions for the LoRa Cloud Geolocation integration with TTS based on the changes done by Semtech.

#### How do you propose to document this?
...

#### Can you do this yourself and submit a Pull Request?

No, @adriansmares

### PR DESCRIPTION
**Pull request by cvetkovski98**
#### Summary
Update the LoRa Cloud integration docs to reflect the updates to the LoRa Could changes and improvements in the Stack.

Closes #944 

#### Changes
- Make new screenshots for the integration
- Extract the LoRa Cloud Portal setup docs under a new sub-section

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/reference/application-packages/lora-cloud-device-and-application-services.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
-title: "LoRa Cloud Device & Application Services"
+title: "LoRa Cloud Modem and Geolocation Services"
```
Comment (nejraselimovic) [inline doc/content/reference/application-packages/lora-cloud-device-and-application-services.md:2]: @adriansmares can you please review this section? **Modem & Geolocation Services** shouldn't be the same as **DAS** afaik. I don't know enough about this application package to be sure that this is completely accurate. I see that some things are changed on LoRa Cloud side.. For example, I suppose that a default server URL isn't https://das.loracloud.com anymore?

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

File: doc/content/integrations/lora-cloud/geolocation-services.md [hunk 2]
```diff
@@ -23,41 +24,33 @@ Three query types are supported in scope of this integration and will be demonst
...
-
-{{< figure src="../geolocation-tokens.png" alt="Pre-configured tokens" >}}
+Before proceding with this guide, make sure you have completed the [LoRa Cloud Setup]({{< ref "/integrations/lora-cloud/portal" >}}) guide.
 
 ## Configure Geolocation Integration
 
 To setup LoRa Cloud Geolocation integration, navigate to your application in {{% tts %}}.
 
-On the left hand menu, select **Integrations &#8594; LoRa Cloud**. Click the **Expand** button next to the **Geolocation**.
+On the left hand menu, select **Integrations &#8594; LoRa Cloud**. Click the **Expand** button next to the **Geolocation services**.
```
[the remark this answers is not in the document]
Comment (cvetkovski98) [reply doc/content/integrations/lora-cloud/geolocation-services.md:39]: Yes, I will make a PR for that and reference it here. The names of the sections have been updated.

File: doc/content/reference/application-packages/lora-cloud-device-and-application-services.md (hunk 1 above)
Comment (adriansmares) [reply doc/content/reference/application-packages/lora-cloud-device-and-application-services.md:2]: Indeed lots of changes happened on the LoRa Cloud side. They stopped separating what used to be DAS (Device and Application Services) from GLS (Geolocation Services). They are now a unified services called Modem and Geolocation services.

File: doc/content/integrations/lora-cloud/portal.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: "LoRa Cloud Setup"
+description: ""
+weight: 1
+---
+
+Before connecting your device to {{% tts %}}, you need to register it in the [LoRa Cloud portal](https://www.loracloud.com/portal) and create an API key.
```
[the remark this answers is not in the document]
Comment (adriansmares) [reply doc/content/integrations/lora-cloud/portal.md:7]: LoRaCloud devices are automatically registered on the first uplink, so I think we should update the text here to mention only the token creation.

File: doc/content/reference/application-packages/lora-cloud-device-and-application-services.md [hunk 4]
```diff
@@ -97,7 +97,7 @@ The package data format is common between both default associations and associat
 
 | Field | Type | Description | Required | Default value |
 |-------|------|-------------|---------|---------------|
-| `token` | `string` | The acces token to be used by the package to submit uplinks to the LoRa Cloud Device & Application Services | Yes | None. |
+| `token` | `string` | The acces token to be used by the package to submit uplinks to the LoRa Cloud Modem and Geolocation Services | Yes | None. |
 | `server_url` | `URL` | The custom base URL to be used for service communication | No | `https://das.loracloud.com`
```
Comment (adriansmares) [inline doc/content/reference/application-packages/lora-cloud-device-and-application-services.md:101]: This should indeed become `mgs.loracloud.com`.

File: doc/content/integrations/lora-cloud/geolocation-services.md [hunk 5]
```diff
@@ -23,41 +24,33 @@ Three query types are supported in scope of this integration and will be demonst
...
+
+Next, copy the token you previously created in LoRa Cloud Modem & Geolocation Services portal.
 
-Next, copy one of the aforementioned tokens from LoRa Cloud Geolocation portal. Paste the token in the **Token** field under the **Set LoRa Cloud token** section. 
+{{< figure src="../copy-token.png" alt="Copy the token from LoRa Cloud" >}}
 
 Choose the **Query type** depending on the type of data you want to use for estimating the location of your device. As mentioned earlier, you can choose **LoRa TOA/RSSI**, **GNSS** or **TOA/WiFi** query type.
```
[the remark this answers is not in the document]
Comment (cvetkovski98) [reply doc/content/integrations/lora-cloud/geolocation-services.md:45]: Added a note.

**Review by adriansmares — APPROVED** [review]

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**